.github/workflows/build.yml

```diff
@@ -54,7 +54,9 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ github.sha }}
-          path: ./*/build/distributions/*.zip
+          path: | 
+            ./*/build/distributions/*.zip
+            sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib/
```

Comment:
@bruno-garcia for some reason its not zipping the folder
tried
`./sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib`
`sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib/`

Reply:
```suggestion
            ./sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib/
```